src/pages/index.md

```diff
@@ -15,6 +15,29 @@ Here's a sample:
 )
 ```
 
+## Why Lisp?
+
+Many people come into our keybase channel and ask us why we chose a 60 year old language as our on chain programming language.
+We chose it due to a few unique features that make it remarkably well suited to the Chia blockchain:
+
+* **Completely sandboxed.** Chialisp has *absolutely no side effects*.  The language needs to be run on half a million computers, so it is important that the program cannot reach out and affect everyone's machines in an unintended way.  A lisp program is *evaluated* and therefore cannot spawn any new processes or interact with the system it is running on.
```

Comment:
"Side effects" usually has a specific technical meaning, and refers to a language being purely functional and without global variables, so perhaps consider different terminology vs "absolutely no side effects". Maybe "Chialisp resource utilization is completely controlled" or something. The idea is it can't create new processes, consume excess memory, use networking, or consume disk space.